homeassistant/components/mqtt/lock.py

```diff
@@ -130,6 +130,7 @@ async def _async_setup_entity(
 class MqttLock(MqttEntity, LockEntity):
     """Representation of a lock that can be toggled using MQTT."""
 
+    _attr_is_locked: bool | None = False
```

Comment:
Can this ever be None?

Reply:
It is unrelated:
It was removed with: #100943